src/structurizr/view/view.py

```diff
@@ -223,7 +247,7 @@ def copy_layout_information_from(self, source: "View") -> None:
                     source_element_view
                 )
 
-        for source_relationship_view in source.relationship_views:
+        for source_relationship_view in source._relationship_views:
```

Comment:
"Private" attributes and methods of other objects should not be accessed.

```suggestion
        for source_relationship_view in source.relationship_views:
```

Reply:
No idea why I changed that!  Reverted.